### PR TITLE
[Term Entry] PHP Math Functions rad2deg()

### PR DESCRIPTION
**Pull request by alexflightlessbird**
### Description

Closes Issue: #2274 
Created rad2deg folder in PHP terms and added rad2deg.md 

### Type of Change

- Adding a new entry
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] I have performed a self-review of my own writing and code.
- [x] My changes generate no new warnings.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/math-functions/terms/rad2deg/rad2deg.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+## Syntax
+```psuedo
+rad2deg(number);
+```
+Where `number` is the radian value to convert to degrees.
+
+The value returned by the function is a `float` type and is the equivalent of `number` in degrees.
+
+## Example
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/rad2deg/rad2deg.md:26]:
```suggestion
## Example

```

File: content/php/concepts/math-functions/terms/rad2deg/rad2deg.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  echo rad2deg(1.71);
+?>
+```
+
+This example results in the following output:
+```shell
+97.975782967371
+```
+
+## Codebyte Example
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/rad2deg/rad2deg.md:40]:
```suggestion
## Codebyte Example

```

File: content/php/concepts/math-functions/terms/rad2deg/rad2deg.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+## Example
+A basic example of the `rad2deg()` function:
+
+```php
+<?php
+  echo rad2deg(1.71);
+?>
+```
+
+This example results in the following output:
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/rad2deg/rad2deg.md:35]:
```suggestion
This example results in the following output:

```

File: content/php/concepts/math-functions/terms/rad2deg/rad2deg.md [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+The **`rad2deg()`** function in PHP converts radians to degrees. It accepts a number (radian value) as an input and returns the corresponding number in degrees.
+
+## Syntax
+```psuedo
+rad2deg(number);
+```
+Where `number` is the radian value to convert to degrees.
+
+The value returned by the function is a `float` type and is the equivalent of `number` in degrees.
```
Comment (dakshdeepHERE) [inline content/php/concepts/math-functions/terms/rad2deg/rad2deg.md:24]:
```suggestion
Where `number` is the radian value to convert to degrees. The value returned by the function is a `float` type and is the equivalent of `number` in degrees.
```

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side

**Comment by alexflightlessbird** [comment]
@SSwiniarski comments have been reviewed and applied!

**Review by SSwiniarski — APPROVED** [review]
@alexflightlessbird I made one small edit and now it looks good for a second review.

**Review by caupolicandiaz — APPROVED** [review]
@alexflightlessbird @SSwiniarski Looks good to merge

**Pull request merged**